### PR TITLE
Settings screen description added to README.md

### PR DESCRIPTION
**Pull request by davyskiba**
This pr adds the `Location logs` section to README.md with a description of log exporting usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,19 @@ desiredInterval provided in milliseconds
 | far  |        2000       |      1000      |
 | near |        1000       |       500      |
 
+## Location logs
+
+To export location logs from the current session, enter the settings screen by clicking the cog icon on the main screen. Logs for each session are stored in two files named by time and date of the first location logged, for example, `14_07_07:06:16_location.log`. Note that location logging starts after adding a trackable.
+- `*_location.log` - contains location reading. Consequent readings are appended to the file as soon as they arrive
+- `*__history.log` - contains location history from the session and is available only after the publisher disconnects
```
Comment (KacperKluka) [inline README.md:99]: I think that the filename does not have double `_`, right? :thinking: 
```suggestion
- `*_history.log` - contains location history from the session and is available only after the publisher disconnects
```

File: README.md [hunk 2]
```diff
@@ -92,6 +92,19 @@ desiredInterval provided in milliseconds
...
+
+There are the following options available on the settings menu:
+- "Close session and save logs" - disconnects current publisher and exports
+- "Export logs" - shares all files present in the log directory inside the app's private storage
+- "Remove logs" - removes all files present in the log directory inside the app's private storage
+- "Start session" - creates a new publisher and connects it to Ably
+- "Restart application" - restarts the application process. Will restart all publishers as well as background services
```
Comment (KacperKluka) [inline README.md:106]: What happens with logs in those cases?

File: README.md [hunk 3]
```diff
@@ -92,6 +92,19 @@ desiredInterval provided in milliseconds
...
 | near |        1000       |       500      |
 
+## Location logs
+
+To export location logs from the current session, enter the settings screen by clicking the cog icon on the main screen. Logs for each session are stored in two files named by time and date of the first location logged, for example, `14_07_07:06:16_location.log`. Note that location logging starts after adding a trackable.
+- `*_location.log` - contains location reading. Consequent readings are appended to the file as soon as they arrive
+- `*__history.log` - contains location history from the session and is available only after the publisher disconnects
+
+There are the following options available on the settings menu:
+- "Close session and save logs" - disconnects current publisher and exports
```
Comment (KacperKluka) [inline README.md:102]: The description seems unfinished, it exports what and where? :thinking:

File: README.md [hunk 4]
```diff
@@ -92,6 +92,19 @@ desiredInterval provided in milliseconds
...
+- "Export logs" - shares all files present in the log directory inside the app's private storage
```
Comment (KacperKluka) [inline README.md:103]: Where does it share them? How can I get them as an app user? :thinking:

**Review by KacperKluka — CHANGES_REQUESTED** [review]
Please add more explanations to the README :bow:

File: README.md (hunk 3 above)
Comment (davyskiba) [reply README.md:102]: right, added 😰

File: README.md (hunk 2 above)
Comment (davyskiba) [reply README.md]: https://github.com/ably/asset-tracking-android-rider-app-demo/pull/37/commits/bb0a8c034b1a3baddb8956671ddf03bcf3861d95

File: README.md (hunk 3 above)
Comment (davyskiba) [reply README.md]: https://github.com/ably/asset-tracking-android-rider-app-demo/pull/37/commits/bb0a8c034b1a3baddb8956671ddf03bcf3861d95

File: README.md (hunk 4 above)
Comment (davyskiba) [reply README.md]: https://github.com/ably/asset-tracking-android-rider-app-demo/pull/37/commits/bb0a8c034b1a3baddb8956671ddf03bcf3861d95

**Review by KacperKluka — APPROVED** [review]
LGTM :+1:

**Pull request merged**